src/UserService.Validation/User/CreateUserRequestValidator.cs

```diff
@@ -50,7 +50,7 @@ public class CreateUserRequestValidator : AbstractValidator<CreateUserRequest>,
             .WithMessage("Middle name must not contain numbers.")
             .Must(x => !SpecialCharactersRegex.IsMatch(x))
             .WithMessage("Middle name must not contain special characters.")
-            .MaximumLength(32)
+            .MaximumLength(45)
             .WithMessage("Middle name is too long.")
```

Comment:
move to 53 line

Reply:
why? here every message comes after verification

Reply:
I mean: .MaximumLength(45).WithMessage("Middle name is too long."). @DarkFelas spoke, what so better

Reply:
done